### PR TITLE
issue/953: make core resilient to theme

### PR DESCRIPTION
**Pull request by oliverfoster**
* added core templates to core from theme
* added foundation less to core from theme
* added icon fonts
* changed grunt to handle nested less folders in core
* changed grunt to handle fonts in core
* changed grunt to handle no theme folder
* added fix to accessibility-state to core

to test:
1. git clone into branch issue/953, or download the branch
1a. remember to do npm install and adapt install on the framework
2. compile and check working
3. delete src/theme folder
4. recompile and check working (see ghastly 1990's no theme version)

#953

### CODE REVIEW

File: src/core/less/theme/colors.less [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+@white: #ffffff;
+@black: #4b4e4f;
+@blue: #7D7D7D;
+@light-grey: #78909c;
+@mid-grey: #546e7a;
+@dark-grey: #37474f;
+@red: #f26c4f;
+@green: #82ca9c;
+
```
Comment (tomgreenfield) [inline src/core/less/theme/colors.less]: Suggest dropping the arbitrary list of 'custom' colours in this non-theme and replacing the subsequent named colours in this file with hex codes?

Comment (oliverfoster) [inline src/core/less/theme/colors.less]: nice idea - remove a layer of abstract from the basic variables

**Comment by lc-thomasberger** [comment]
@oliverfoster 
when I delete the src/theme folder, it throws an Error: template is not a function. app.js Line 37.
Looks like Handlebars.templates['loading'] is not defined.

**Comment by oliverfoster** [comment]
let me just make sure i can replicate, but the code looks ok... give me an hour or so

**Comment by oliverfoster** [comment]
holding this pr until we have a better understand of how to reduce / expand the core theme to suit everyone's needs.

**Pull request closed without merging**